blackboxopt/optimization_loops/sequential.py

```diff
@@ -52,6 +53,8 @@ def run_optimization_loop(
             trace in the evaluation's `stacktrace` attribute. Set to True if there are
             spurious errors due to e.g. numerical instability that should not halt the
             optimization loop.
+        post_evaluation_callback: Reference to a callable that is invoked after each
+            evaluation.
```

Comment:
Would it make sense if docstring here also reflects that the evaluation is the only argument to the callable? It is technically covered by the typehints, but maybe a half-sentence here wouldn't hurt?